<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-13T13:59:14 IST" name="Default suite" finished-at="2022-10-13T14:01:13 IST" duration-ms="118451">
    <groups>
    </groups>
    <test started-at="2022-10-13T13:59:14 IST" name="Default test" finished-at="2022-10-13T14:01:13 IST" duration-ms="118451">
      <class name="test.ThreeTest">
        <test-method signature="threeTest()[pri:0, instance:test.ThreeTest@353352b6]" started-at="2022-10-13T13:59:15 IST" name="threeTest" finished-at="2022-10-13T13:59:51 IST" duration-ms="36068" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- threeTest -->
      </class> <!-- test.ThreeTest -->
      <class name="test.LoginTest">
        <test-method is-config="true" signature="openApplication()[pri:0, instance:test.LoginTest@7d9d0818]" started-at="2022-10-13T13:59:51 IST" name="openApplication" finished-at="2022-10-13T14:00:02 IST" duration-ms="11154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:test.LoginTest@7d9d0818]" started-at="2022-10-13T14:00:02 IST" name="login" data-provider="getLoginData" finished-at="2022-10-13T14:00:08 IST" duration-ms="5635" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[kundansinghreddy@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Kundan@26]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Successfull]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="closerMethod()[pri:0, instance:test.LoginTest@7d9d0818]" started-at="2022-10-13T14:00:08 IST" name="closerMethod" finished-at="2022-10-13T14:00:08 IST" duration-ms="147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closerMethod -->
        <test-method is-config="true" signature="openApplication()[pri:0, instance:test.LoginTest@7d9d0818]" started-at="2022-10-13T14:00:08 IST" name="openApplication" finished-at="2022-10-13T14:00:49 IST" duration-ms="40891" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:test.LoginTest@7d9d0818]" started-at="2022-10-13T14:00:49 IST" name="login" data-provider="getLoginData" finished-at="2022-10-13T14:00:54 IST" duration-ms="5399" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[kundansinghreddy@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Kundan26]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Failure]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="closerMethod()[pri:0, instance:test.LoginTest@7d9d0818]" started-at="2022-10-13T14:00:54 IST" name="closerMethod" finished-at="2022-10-13T14:00:54 IST" duration-ms="432" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closerMethod -->
      </class> <!-- test.LoginTest -->
      <class name="test.FourTest">
        <test-method signature="fourTest()[pri:0, instance:test.FourTest@4007f65e]" started-at="2022-10-13T14:00:54 IST" name="fourTest" finished-at="2022-10-13T14:01:04 IST" duration-ms="9883" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at test.FourTest.fourTest(FourTest.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fourTest -->
        <test-method is-config="true" signature="closingBrowser()[pri:0, instance:test.FourTest@4007f65e]" started-at="2022-10-13T14:01:04 IST" name="closingBrowser" finished-at="2022-10-13T14:01:04 IST" duration-ms="116" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingBrowser -->
      </class> <!-- test.FourTest -->
      <class name="test.TwoTest">
        <test-method signature="twoTest()[pri:0, instance:test.TwoTest@1a245833]" started-at="2022-10-13T14:01:04 IST" name="twoTest" finished-at="2022-10-13T14:01:13 IST" duration-ms="8485" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoTest -->
      </class> <!-- test.TwoTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
